tests/test_tacacs_server_host_style2.rb

```diff
@@ -0,0 +1,219 @@
+#
```

Comment:
As discussed, we need to either:
a) replace the existing test_tacacs_server_host minitest with this new minitest (without losing significant test coverage)
b) make sure the old minitest skips for XR